packages/venia-ui/lib/components/CreateAccount/createAccount.js

```diff
@@ -10,6 +10,7 @@ import CREATE_CART_MUTATION from '../../queries/createCart.graphql';
 import GET_CART_DETAILS_QUERY from '../../queries/getCartDetails.graphql';
 import GET_CUSTOMER_QUERY from '../../queries/getCustomer.graphql';
 import SIGN_IN_MUTATION from '../../queries/signIn.graphql';
+import { mergeCarts as MERGE_CARTS_MUTATION } from '../../queries/mergeCarts.gql';
```

Comment:
Should we just rename the export so you don't have to alias?

Reply:
heh, yes.